<?xml version="1.0"?>

<launch>
    <arg name="output"                      default="screen"/> 
    <arg name="required"                    default="true"/>
    <arg name="respawn"                     default="false"/>

    <node pkg="mower_location" type="mower_location_node" name="mower_location" 
          output="$(arg output)" required="$(arg required)" respawn="$(arg respawn)">

        <param name="devname"                   value="/dev/mower_gnss"/>
        <param name="baudrate"                  value="115200"/>
        <param name="gnss_topic"                value="/sensor_msgs/NavSatFix"/>
        <param name="imu_topic"                 value="/sensor_msgs/Imu"/>
        <param name="mower_gnss_topic"          value="/mower/gnss"/>
        <param name="mower_odom_topic"          value="/mower/odom"/>
        <param name="mower_chassis_state_topic" value="/mower/chassis_state"/>
        <param name="mower_map_meta_topic"      value="/mower/map_meta"/>

		<param name="mower_aruco_detect_topic"  value="/mower/aruco_detect"/>
		<param name="mower_home_update_topic"  	value="/mower/home_update"/>

    </node>
</launch>


